project.mml

```diff
@@ -1411,6 +1411,22 @@ Layer:
     properties:
       cache-features: true
       minzoom: 12
+  - id: stop-positions
+    geometry: point
+    <<: *extents
+    Datasource:
+      <<: *osm2pgsql
+      table: |-
+        (SELECT
+          way, 
+          tags->'public_transport' as public_transport,
+          ref,
+          tags->'local_ref' as local_ref
+        FROM planet_osm_point
+        WHERE (tags->'public_transport' = 'stop_position') AND (tags->'train' = 'yes') AND ((ref IS NOT NULL) OR ((tags->'local_ref') IS NOT NULL))
+        ) AS stop_positions
```

Comment:
This needs an ordering, otherwise it will be inconsistent between adjancent tiles

Reply:
Thanks, I tried some ordering but it didn't change anything. What should I order by?

Reply:
Looks like the cut off labels are a general issue of the docker setup and are not caused by this pullrequest (see #4808)

Reply:
Ordering should be by something that makes sense for the priority of the features and the very last thing should be osm_id

Reply:
I tried different orderings two weeks ago and in the end ended up with osm_id which should garante a consistent ordering. Today I removed the entier order by section as I don't think that the ordering has anything to do with the rendering artifacts (see #4808 ).

Reply:
These are collisions caused by one tile not considering what is rendered in the next tile. Ordering could only contribute to solving collisions between two track numbers, but the observed collisons are between track numbers and other objects.

I don't think that this issue will occur in the productive tile rendering for osm.org as this appears to be a problem fo the docker test setup, but it would be good to double check this of course.